packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -434,32 +451,62 @@ const VirtualListBaseFactory = (type) => {
 				if (SpotlightAccelerator.processKey(ev, nop)) {
 					ev.stopPropagation();
 				} else {
-					const {repeat, target} = ev;
-					const index = getNumberValue(target.dataset.index);
-					const {isBackward, isForward, isLeftMovement, isRightMovement, isWrapped, nextIndex} = this.getNextIndex({index, keyCode, repeat});
-
-					if (nextIndex >= 0) {
-						ev.preventDefault();
-						ev.stopPropagation();
-						this.onAcceleratedKeyDown({isWrapped, keyCode, nextIndex, repeat, target});
+					const {repeat} = ev;
+					const {dimensionToExtent, isPrimaryDirectionVertical} = this.uiRefCurrent;
+					const isScrollButton = target.classList.contains(scrollbarCss.scrollButton);
```

Comment:
I concerned that we started adding some code related with scrollbar in `VirtualListBase` including `scrollAndFocusScrollbarButton`. That might increase the complexity of the `VirtualListBase`. If possible, it would be better to add the code related with scrollbar in not `VirtualListBase` but `ScrollableBase`.